contracts/src/test/kotlin/com/r3/developers/apples/contracts/AppleStampContractIssueCommandTest.kt

```diff
@@ -0,0 +1,95 @@
+package com.r3.developers.apples.contracts
+
+import com.r3.corda.ledger.utxo.testing.ContractTest
+import com.r3.corda.ledger.utxo.testing.buildTransaction
+import com.r3.developers.apples.states.AppleStamp
+import net.corda.v5.ledger.utxo.Command
+
+import org.junit.jupiter.api.Test
+import java.security.PublicKey
+import java.util.UUID
+
+class AppleStampContractIssueCommandTest: ContractTest() {
+    private val outputAppleStampStateId: UUID = UUID.randomUUID()
+    private val outputAppleStampStateStampDesc: String = "Can be exchanged for a single basket of apples"
+    private val outputAppleStampStateIssuer: PublicKey = bobKey
+    private val outputAppleStampStateHolder: PublicKey = daveKey
+    private val outputAppleStampStateParticipants: List<PublicKey> = listOf(bobKey,daveKey)
+    internal val outputAppleStampState: AppleStamp = AppleStamp(
+        outputAppleStampStateId,
+        outputAppleStampStateStampDesc,
+        outputAppleStampStateIssuer,
+        outputAppleStampStateHolder,
+        outputAppleStampStateParticipants
+    )
+
+    @Test
+    fun happyPath(){
+        val transaction = buildTransaction {
+            addCommand(AppleCommands.Issue())
+            addOutputState(outputAppleStampState)
+            addSignatories(outputAppleStampStateParticipants)
+        }
+        assertVerifies(transaction)
+    }
+
+    @Test
+    fun outputContractStateSizeNotOne(){
+        val transaction = buildTransaction {
+            addCommand(AppleCommands.Issue())
+            addOutputState(outputAppleStampState)
+            addOutputState(outputAppleStampState)
+            addSignatories(outputAppleStampStateParticipants)
+        }
+        assertFailsWith(
+            transaction,
+            "This transaction should only have one AppleStamp state as output"
```

Comment:
Not a must, but you might find this relevant: https://github.com/corda/CSDE-cordapp-template-kotlin/blob/dcb0a2b77a170065558820cc38468f57f551225f/contracts/src/main/kotlin/com/r3/developers/csdetemplate/utxoexample/contracts/ChatContract.kt#L11-L28

Reply:
I agree that this should be the standard.

I wanted to apply this practice in a future PR, but I chose to stick with the current way to match what is written in the guide.

The idea is to sync a PR with the document changes to the tutorial so that no one gets mixed up